OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -6,14 +6,25 @@
 
 #include "FunctionUtils.h"
 #include "OrbitBase/Logging.h"
-
 #include "absl/flags/flag.h"
 
 ABSL_DECLARE_FLAG(uint16_t, sampling_rate);
 ABSL_DECLARE_FLAG(bool, frame_pointer_unwinding);
 
 using orbit_client_protos::FunctionInfo;
 
+static CaptureOptions::InstrumentedFunction::FunctionType
+IntrumentedFunctionTypeFromOrbitType(FunctionInfo::OrbitType orbit_type) {
+  switch (orbit_type) {
+    case FunctionInfo::kOrbitTimerStart:
+      return CaptureOptions::InstrumentedFunction::kTimerStart;
+    case FunctionInfo::kOrbitTimerStop:
+      return CaptureOptions::InstrumentedFunction::kTimerStop;
+    default:
```

Comment:
There are several other values for `OrbitType`, should they all be mapped to `kRegular`?

Reply:
Mmmh, this is out of the scope of this change and complicates things, but why do you need the uretprobe for types like `kOrbitTrackInt`? Don't you only need the parameters?

Reply:
We have the duplicate uprobes on thread migration issue for which we have a workaround that relies on uprobe-uretprobe pairs.